libs/core/src/global/styles/tokens/semantic/semantic.json

```diff
@@ -372,7 +372,7 @@
   "body-xs": {
     "value": {
       "fontWeight": "{font-weight.normal}",
-      "fontSize": "{font-size.075}",
+      "fontSize": "{font-size.75}",
       "lineHeight": "{line-height.100}",
       "fontFamily": "{font-family.greet}"
```

Comment:
Also may not be related to this PR, but I think greet is only applied to headings.

Reply:
I'm actually just going to remove these semantic tokens since they aren't in use at the moment and I will be redoing them soon.